pixelmod/src/main/java/io/pixelmod/IDecalRenderer.kt

```diff
@@ -0,0 +1,7 @@
+package io.pixelmod
+
+import android.graphics.Bitmap
+
+interface IDecalRenderer {
```

Comment:
I don't like the `I` prefix. feels like C++

Reply:
This is pretty common in any OOP languages.

[https://github.com/AndroidSDKSources/android-sdk-sources-for-api-level-28/blob/809dd8b7822332cdbc68939d777eb70a6951908a/android/content/IContentProvider.java](url)

Reply:
hehe